css/index.css

```diff
@@ -172,4 +172,38 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+@media(max-width:768px){
+    .container{
+        width: 100%;
+        padding-right:20px;
+        padding-left: 20px;
+    }
+    header{
+        width:100%;
+        display:flex;
+        flex-direction: column-reverse;
+        align-items: center; 
+    }
+    header nav{
+        width:100%;
+    }
+    header .logo{
+        margin-right:300px;
+    }
+    .cta{
+        display:flex;
+        justify-content: center;
+    }
+    .cta img {
+        display:none;
+    }
+    
+    .main-content .top-content {
+        width: 100%;
+    }
+    .main-content .middle-img{
+        width:100%
+    }    
+   
```

Comment:
Let me know when you implement the @media max 400 so I can offer another review.